js/number-isfinite/index.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "Number.isFinite"
+tags:
+  - doka
+authors:
+  - windrushfarer
+---
+
+## Кратко
+
+Метод `Number.isFinite` позволяет проверить является ли переданное в аргументе число конечным. Конечным числом можно считать любое число, кроме бесконечности. В JavaScript бесконечность можно выразить только с помощью глобального `Infinity` или полей `Number.POSITIVE_INFINITY` и `Number.NEGATIVE_INFINITY`.
```

Comment:
NaN еще тоже считается бесконечным

Reply:
Здесь речь ведь идет именно про бесконечность. `NaN` это ведь просто не-число :) Не хочется вводить в заблуждение

Reply:
Мне просто кажется странным не сказать про нан в кратком описании. Его должно быть достаточно, чтобы понять функцию.

Может сказать, что бесконечными считаются вот эти значения: 1,2,3.

На мдн смотри как:
> You can use this function to determine whether a number is a finite number. The isFinite function examines the number in its argument. If the argument is NaN, positive infinity, or negative infinity, this method returns false; otherwise, it returns true.